crypto/hash/null_auth.c

```diff
@@ -81,44 +86,51 @@ static srtp_err_status_t srtp_null_auth_alloc (srtp_auth_t **a, int key_len, int
     return srtp_err_status_ok;
 }
 
-static srtp_err_status_t srtp_null_auth_dealloc (srtp_auth_t *a)
+static srtp_err_status_t srtp_null_auth_dealloc(srtp_auth_t *a)
 {
     extern const srtp_auth_type_t srtp_null_auth;
 
     /* zeroize entire state*/
-    octet_string_set_to_zero(a, sizeof(srtp_null_auth_ctx_t) + sizeof(srtp_auth_t));
+    octet_string_set_to_zero(a, sizeof(srtp_null_auth_ctx_t) +
+                                    sizeof(srtp_auth_t));
```

Comment:
This line looks like unusual indentation.

Reply:
I think it sees the two sizeof call as one operation, ie no "," and so when breaking an operation on to two lines it will indent.